daemon/oci_windows.go

```diff
@@ -471,7 +491,7 @@ func readCredentialSpecFile(id, root, location string) (string, error) {
 	}
 	bcontents, err := ioutil.ReadFile(full)
 	if err != nil {
-		return "", fmt.Errorf("credential spec '%s' for container %s as the file could not be read: %q", full, id, err)
+		return "", fmt.Errorf("credential spec for container %s could not be read from file %q: %q", id, full, err)
```

Comment:
I saw @dperny was working on some of those, but while we're changing this, we should change these to;

```suggestion
		return "", errors.Wrapf(err, "credential spec for container %s could not be read from file %q", id, full)
```

Reply:
Good call, doing :)